### ISSUE TITLE
Last chart version in helm-repo is 5.0.3

### ISSUE THREAD

**Issue by joggeli34** [emqx/emqx#8953]
### What happened?

The last released version currently is 5.0.7, but in the helm-chart repo I find the 5.0.3 as the latest one. So the helm-chart is probably not deployed?

```
PS C:\git\helm-charts> helm search repo emqx
NAME                    CHART VERSION   APP VERSION     DESCRIPTION                              
emqx/emqx               5.0.3           5.0.3           A Helm chart for EMQX
emqx/emqx-ee            4.4.8           4.4.8           A Helm chart for EMQ X
emqx/emqx-operator      2.0.1           2.0.1           A Helm chart for EMQX Operator Controller
emqx/kuiper             0.9.0           0.9.0           A lightweight IoT edge analytic software
```

### What did you expect to happen?

Newest chart version should match the newest helm version

### EMQX version

5.0.7

**Comment by terry-xiaoyu** [comment]
Hi, this will be fixed in emqx v5.0.8

**Comment by luzhkovvv** [comment]
Hi! v5.0.8 was released several days ago, but the repository still hosts v5.0.3 only.

**Comment by joggeli34** [comment]
Any news about this topic? We are currently migrating to emqx 5.0 and it would be greate to use the newest helm chart with the newest docker image.

**Comment by luzhkovvv** [comment]
The issue is not resolved still, not stale.

**Comment by FinitelyFailed** [comment]
Any news on this? We are also trying to migrate to emqx 5.0.

**Comment by HJianBo** [comment]
I just learned how to update it. Thanks for the feedback guys.

**Comment by FinitelyFailed** [comment]
Hi! Has the issue been resolved?

**Comment by HJianBo** [comment]
Hi @FinitelyFailed It has been fixed by https://github.com/emqx/emqx/pull/8921. So it should work once v5.0.9 is released this weekend

**Comment by FinitelyFailed** [comment]
@HJianBo nice! Looking forward to test 5.0.9 next week then. :)

**Comment by ewilken** [comment]
Any news on this?

**Comment by HJianBo** [comment]
Need your help @id It looks like there are some errors in the CI. https://github.com/emqx/emqx/actions/runs/3311249946/jobs/5466460320

**Comment by ewilken** [comment]
Unfortunately the problem seems to still persist. 😕 So this issue probably shouldn't get stale.

**Comment by luzhkovvv** [comment]
Please reopen the issue. After 5.0.10 release helm chart is still 5.0.3 in the repository, "Upload release assets" workflow failed to run at all because of syntax errors https://github.com/emqx/emqx/actions/runs/3430641285. @id

**Comment by joggeli34** [comment]
It is still missing, even after the workflow syntax fix (https://github.com/emqx/emqx/commit/155d4a98184b189a45c22db37198f5e54d9be430)

**Comment by tommyminds** [comment]
Any update on when this will be fixed? 5.0.3 is currently still the chart in the repository.

**Comment by joggeli34** [comment]
Now, 5.0.11 is in the public repo, thanks
